customisation/customisation.md

```diff
@@ -5,9 +5,9 @@ title: Customisation
 description: Machine translation customisation
 ---
 
-{% assign customizable_apis = site.data.apis | where: "customization_languages" | where "glossary" | concat: site.data.apis | where "formality" | concat: site.data.apis | where "adaptive" %}
+{% assign customizable_apis = site.data.apis | where: "customization_languages", true | where: "glossary", true | concat: site.data.apis | where: "formality", true | concat: site.data.apis | where: "adaptive", true %}
```

Comment:
Seems like there is a `concat: site.data.apis |` missing after `where: "customization_languages", true |`

Reply:
But it won't be any nicer.